### ISSUE TITLE
[Intermittent][MSF4J][Windows] MSF4J not going to start with an error

### ISSUE THREAD

**Issue by Dilshani** [wso2/product-ei#475]
**Environment**
Operating Systems : Windows 7
Java Version : JDK1.8
Packs : EI610 latest pack
EI Setup: Standalone
DB - H2 (Default)

**Steps to reproduce**
Start msf4j in windows. 
(Note: This issue comes when msf4j start up parallel with other server profiles)

**Expected Result**
MSF4J should be started

**Actual Result**
Getting an error:

```
[2017-04-04 05:39:52,609]  INFO {org.wso2.carbon.transport.http.netty.list
ener.HTTPTransportListener} - HTTP Interface default starting on host  0.0.0.0 a
nd port 9090
Exception in thread "Timer-0" java.lang.NullPointerException
        at org.wso2.carbon.kernel.internal.startupresolver.StartupOrderResolver$
1.run(StartupOrderResolver.java:170)
        at java.util.TimerThread.mainLoop(Timer.java:555)
        at java.util.TimerThread.run(Timer.java:505)

```

**Comment by erandacr** [comment]
This is a known issue in Kernel 5.1.0, it will get fixed[1] in kernel 5.2.0 which is not yet released. 

[1].https://wso2.org/jira/browse/CARBON-15996

**Comment by nandika** [comment]
Fixed with kernel 5.1.1 update.

**Comment by erandacr** [comment]
Fixed with https://github.com/wso2/product-ei/pull/487
